clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -808,6 +808,31 @@ else if (max == null)
         }
     }
 
+    public static class ValidList implements Validator {
```

Comment:
Added the test to ConfigDefTest as that is where the other Validators are tested